ir/v1.cpp

```diff
@@ -131,14 +133,10 @@ unsigned IR::Primitive::inferOperandTypes() const {
 const IR::Type *IR::Primitive::inferOperandType(int operand) const {
     if (name == "truncate")
         return IR::Type::Bits::get(32);
-    if ((name == "count" || name == "execute_meter") && operand == 1) {
-        if (auto obj = operands[0]->to<IR::GlobalRef>()) {
-            if (auto tbl = obj->obj->to<IR::Stateful>()) {
-                if (tbl->instance_count > 0) {
-                    int width = ceil_log2(tbl->instance_count);
-                    return IR::Type::Bits::get(width); } } } }
-    if (name.startsWith("execute_stateful") && operand == 1) {
-            return IR::Type::Bits::get(32); }
+    if ((name == "count" || name == "execute_meter") && operand == 1)
+        return IR::Type::Bits::get(32);
```

Comment:
Is there a reason to remove the calculation of the correct size and use a fixed, incorrect size?